### ISSUE TITLE
Unify configuration documentation

### ISSUE THREAD

**Issue by estan** [crossbario/crossbarwww#53]
For the configuration documentation, it would be great if the full structure, including all available parameters, were described at a single place and in a consistent format. As it is now, the configuration documentation is spread over a rich set of links under Administration / Configuration at http://crossbar.io/docs/ . Each linked page is based mostly on a lot of text interspersed with examples, some of them have a table titled "The available parameters for XXX are:" at the end.

Examples are good, don't get me wrong. The pages should definitely stay. But at the moment there's a lot of hunting involved if you want a full picture of the config. I think there should be a more terse, formal and exhaustive description of what the JSON file can contain somewhere. That one could include links to the detailed descriptions. Since it's JSON, isn't there some standard way of documenting the (informal) schema for it?

I think such a brief / formal description could be easier to keep up to date as well. Looking at the snippets at some of the pages, e.g. http://crossbar.io/docs/WebSocket-Transports/ , I can't reconcile the config snippet I see there with what I get with `crossbar init`, so I think it may be outdated.

**Comment by goeddea** [comment]
Would a large config file which contains every option set help?

Regarding  http://crossbar.io/docs/WebSocket-Transports/  - this is current. The displayed config is for a WebSocket endpoint, wheras the default config you get with `crossbar init` uses a WebSocket transport on a path of a Web transport.

**Comment by estan** [comment]
Yep, a large config would help I guess, but I'm not sure it would be the best thing. If it is to show all allowed values of parameters, I think it would explode in size from all the combinations. It would also be good if there, for each config options, could be links to where more info is to be found in the current docs. So I think a brief page that includes all config options and their allowed values might be better.
